public/js/queries/get_nnearest.js

```diff
@@ -1,7 +1,7 @@
 module.exports = {
   render: _ => `
-  
   SELECT
+    *,
```

Comment:
I don't understand much of it, but is this necessary? doesn't it mean it selects everything?

Reply:
Actually, I need it to mock the draft pin. I don't need everything just the sentiment and status but this function is used by other layers so to not break it I'm getting all the data. This is already merged I think (https://github.com/Commonplace-Digital-Ltd/xyz/commit/7830ea7f901b440fb195ae071baa16fa4a20857e)